src/components/common/Button.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { cn } from '@/utils/cn';
+/**
+ * Button 컴포넌트
+ *
+ * @param {string} btnText - 버튼에 표시할 텍스트
+ * @param {React.ReactNode} icon - 버튼에 포함될 아이콘
+ * @param {'xl' | 'l' | 'm' | 's' | 'xs'} [size='xl'] - 버튼의 크기
+ * @param {'gray' | 'pink'} [color='pink'] - 버튼 색상
+ * @param {() => void} onClick - 버튼 클릭 시 실행할 함수
+ * @param {string} [className=''] - 추가적인 스타일
+ * @param {boolean} [border=false] - 버튼에 border를 부여할지(border를 주고싶으면 props를 넘길때 border라고만 적기)
+ * @param {boolean} [disabled=false] - 버튼 비활성화 여부(props를 줄때 disabled라고만 적기)
+ * @param {boolean} [rounded=false] - 버튼이 둥근형태일지 여부(props를 줄때 rounded라고만 적기)
+ *
+ * @returns {JSX.Element} <Button/>컴포넌트
+ * 
+ * 
+ * 예시
+ *   <Button
+        className='mt-4 mb-4 ml-4' //추가 클래스네임
+        btnText='확인'
+        color='pink'
+        size='xs'
+        disabled
+        rounded
+        onClick={handleTest}
+      />
+
+
+        <Button
+        className='mt-4 mb-4 ml-4'
+        btnText='확인'
+        color='gray'
+        size='xs'
+        rounded
+      />
+ */
+const Button = ({
+  btnText, //버튼 텍스트
+  icon, //아이콘 유무
+  size = 'xl', //버튼 사이즈
+  color = 'pink', //색상
+  onClick, //onclick함수
```

Comment:
props 함수도 기본 값을 지정해주면 예기치 못한 상황을 방지할 수 있을 거 같습니다!

```suggestion
  onClick: () => { console.log('버튼 클릭') }
```